fcrepo-http-commons/src/test/java/org/fcrepo/http/commons/exceptionhandlers/ConcurrentUpdateExceptionMapperTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree.
+ */
+package org.fcrepo.http.commons.exceptionhandlers;
+
+import org.fcrepo.config.FedoraPropsConfig;
+import org.fcrepo.kernel.api.exception.ConcurrentUpdateException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+import javax.inject.Inject;
+import javax.ws.rs.core.Response;
+
+import static javax.ws.rs.core.Response.Status.CONFLICT;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.util.ReflectionTestUtils.setField;
+
+/**
+ * ConcurrentUpdateExceptionMapperTest
+ *
+ * @author dan.field@lyrasis.org
+ */
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class ConcurrentUpdateExceptionMapperTest {
+
+    @Inject
```

Comment:
Not necessary as you are actually instantiating the class in the `setUp()` method